pkg/osvscanner/vulnerability_result.go

```diff
@@ -79,7 +79,8 @@ func buildVulnerabilityResults(
 
 		// For Debian-based ecosystems, mark unimportant vulnerabilities within the package.
 		// Debian ecosystems may be listed with a version number, such as "Debian:10".
-		if strings.HasPrefix(pkg.Package.Ecosystem, string(models.EcosystemDebian)) {
+		if strings.HasPrefix(pkg.Package.Ecosystem, string(models.EcosystemDebian)) ||
+			strings.HasPrefix(pkg.Package.Ecosystem, string(models.EcosystemUbuntu)) {
```

Comment:
Please use the osvschema Ecosystems here

Reply:
Updated